### DISCUSSION TITLE
Starting up with section 4 with windows and wsl ubutu users

### DISCUSSION THREAD

**Discussion by soyaya** [Cyfrin/security-and-auditing-full-course-s23#62] [Show and tell]
make sure you get the exact path for line 13 after cloning the repo.  remove :; rm -rf /mnt/c/pr0/password/puppy2/4-puppy-raffle-audit/gitmodules && rm -rf /mnt/c/pr0/password/puppy2/4-puppy-raffle-audit/git/modules/* && rm -rf lib && touch .gitmodules && git add . && git commit -m "modules" before you run 'make'. if not you might get this error: [Makefile:13: remove] Error 1

**Comment by cromewar** [comment]
Thanks for this @soyaya we will add this as an annotation can you please tell us which specific lesson on updraft you refer to? or YouTube time stamp?

**Reply by soyaya** [reply]
@cromewar thank you; it can be found in Lesson 3: Phase 1: Scoping. time stamp 1:50 when you clone the repo and run make.
